src/App/shared/components/SearchForm.js

```diff
@@ -6,6 +6,7 @@ const StyledSearchForm = styled(Form)`
   &&& {
     font-size: 1.1rem;
     margin: 0 auto;
+    margin-bottom: 0.8em;
```

Comment:
This is meant to be "close" to the search box.... I think we could try `0.3em` in this case. Or maybe just not having any margin and leave at it is... :)

Reply:
0.3 was way too close imho.

Reply:
0.5?

Reply:
ok.. so then please add this margin in the `SearchOptions` component instead. 
https://github.com/vespa-engine/cord-19/blob/jonmv/paginate-search-results/src/App/Search/SearchOptions.js#L7

Replace `margin-top: 2px;` with:

```
margin-top: 0.5em;
margin-bottom: 0.5em;
```

Reply:
Ah, see reply in main thread.